src/main/resources/application.properties

```diff
@@ -1,4 +1,9 @@
 springdoc.api-docs.path=/api-docs
 springdoc.swagger-ui.path=/swagger-ui
+server.port=13000
 
-
+spring.jpa.hibernate.ddl-auto=update
+spring.datasource.url=jdbc:mysql://noucake.ddns.net:3306/aztube
+spring.datasource.username=root
+spring.datasource.password=rootpw
```

Comment:
Passwords shouldnt be included inside the repository

Reply:
https://aztube.atlassian.net/browse/AX-106